bloom-housing/bloom
# 2852/new flagged management page

## Issue Overview

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/2852)


## Description
This pr handles 2 pieces:
1) handles the new front end for the flagged records (the table/general layout)
2) makes modifications to the flagged set, and applications table.
mainly it removes the status flag at the flagged set layer, and adds something similar to the applications table 

## How Can This Be Tested/Reviewed?
after a reseed there will be data present for flagged applications (duplicates)

go in and navigate to a listing's detail page, from there head the applications tab, and from there select pending reviews via the side nav (up to this point the work was done in other prs)

select one of these flagged sets from the table, you should be navigated to a new table of the dupicates. You should be able to check some number of them, and the data on this table should look like the mock attached to the ticketo

## Checklist:

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
